app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/countries_overview_rv"
```

Comment:
Actually I didn't put a lot of effort on this because for me it's also ok both ways! 
I would say:
- Camel case in XML if we are using the synthetics directly
- Underscore notation otherwise, so we can know if we are referencing the `R` id or the kotlin name. For this one also, if we implement the `Binding` library (I was gonna do it but wanted to close this "fast"), it automatically converts underscore notation into camelCase :)

Reply:
Nice!!! So it's ok for me this way too. 😄 👍